ui/imports/shared/views/AssetsView.qml

```diff
@@ -159,7 +162,24 @@ Control {
 
                     objectName: "cmbTokenOrder"
                     hasCustomOrderDefined: root.customOrderAvailable
-
+                    Binding {
+                        target: sortOrderComboBox
+                        property: "currentIndex"
+                        value: {
+                            let id = sortOrderComboBox.indexOfValue(root.sortValue)
```

Comment:
In my setup it initially evaluates to `-1` because the binding is evaluated before the model is set. Adding `sortOrderComboBox.count` helps to trigger it once again when the model is set.

![Screenshot from 2024-09-09 11-53-01](https://github.com/user-attachments/assets/c9eed107-65c3-4ddf-a8f1-58a98b428c9d)

Reply:
It would be also nice to cover the added functionality to the `AssetsView`'s sb page.